spec/lib/vmdb/loggers/fog_logger_spec.rb

```diff
@@ -2,7 +2,7 @@
   before do
     @log_stream = StringIO.new
     @log = described_class.new(@log_stream)
-    @log.level = VMDBLogger::DEBUG
+    @log.level = ManageIQ::Loggers::Base::DEBUG
```

Comment:
I'm not sure why the original code retrieved log levels from VMDBLogger... This line can just be `Logger::DEBUG`